Added first version of models including serialization

maarten551 opened the pull request:
Added websocket service including generic abstract class
Added very simple and crude lobby-overview
Bugfix: fixed the css to scss

maarten551 committed:
Added first version of models including serialization
Added websocket service including generic abstract class
Added very simple and crude lobby-overview
Bugfix: fixed the css to scss

eclipticrick commented on src/app/app.component.ts:
```diff
@@ -23,6 +25,13 @@ export class AppComponent {
     this.rxStompService.watch('/chat').subscribe((message: Message) => {
```
Should this subscription be closed? in `ngOnDestroy` maybe?

maarten551 replied:
Is indeed an idea, maybe close when no-one is subscribed and re-open is when someone is subscribing again.

However, was to lazy to implement it atm.

eclipticrick reviewed (approved):
Approved!

mbobbeldijk committed:
Added better logic for players
And a bunch of other stuff

maarten551 committed:
Added error support (with toaster)
It's now possible to join a lobby

maarten551 committed:
Cleaned up the lobby system
It's not possible to retrieve messages for your specific match
Also added the first few things for actually creating the game

maarten551 committed:
Added game-session component for showing the actual game
It's not possible to make a move!!!
